internal/core/input.rs

```diff
@@ -151,6 +131,29 @@ pub mod key_codes {
     macro_rules! declare_consts_for_special_keys {
        ($($char:literal # $name:ident # $($_qt:ident)|* # $($_winit:ident)|* ;)*) => {
             $(pub const $name : char = $char;)*
+
+            #[allow(missing_docs)]
+            #[derive(Debug, Copy, Clone, PartialEq)]
+            #[non_exhaustive]
+            #[repr(C)]
+            /// A keyboard key
```

Comment:
The documentation should mention a bit more about what this struct is for.
Especially that it can be used to create a WindowEvent and maybe a code example to do that.

Reply:
done